keyboards/ergodash/mini/keymaps/custom/keymap.c

```diff
@@ -0,0 +1,132 @@
+
+
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 16
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+};
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
```

Comment:
These two keycodes are already defined in QMK, there's no need to redefine them here :)

Reply:
I`m so sorry to had a misstake of pull request base repository.
I close this request.

Reply:
I`m so sorry to had a misstake of pull request base repository.
I close this request.